tests/v2/validation/provisioning/rke1/README.md

```diff
@@ -227,4 +228,59 @@ For custom clusters, the below config is needed, only AWS/EC2. It is important t
     "awsUser": "ubuntu",
     "volumeSize": 50
   },
+```
+
+## Advanced Settings
+This encapsulates any other setting that is applied in the cluster.spec. Currently we have support for:
+* cluster agent customization 
+* fleet agent customization
+
+Please read up on general k8s to get an idea of correct formatting for:
+* resource requests
+* resource limits
+* node affinity
+* tolerations
+
+```json
+"advancedOptions": {
+    "rke1ClusterAgentCustomization": { // change this to rke1FleetAgentCustomization for fleet agent
```

Comment:
This is how it is defined in Rancher's network traffic, correct? Just validating as RKE2/K3S don't have the cluster type in front of it and `config.go` doesn't have `rke1ClusterAgentCustomization`.

I am sure it's network traffic, but just wanted to validate 😄

Reply:
oh thanks, I need to update the readme after the requests from Ugur/Israel, will do that next.